commit b7be91e96ee43998e23067a817e53f36f5692add
Author: q66 <daniel@octaforge.org>
Date:   Mon Oct 10 18:27:04 2022 +0000

    add missing kconfig variables so that refresh does not drop them

diff --git a/board/boundary/nitrogen8m_som/Kconfig b/board/boundary/nitrogen8m_som/Kconfig
index 3d71f9e..ae7f7af 100644
--- a/board/boundary/nitrogen8m_som/Kconfig
+++ b/board/boundary/nitrogen8m_som/Kconfig
@@ -9,6 +9,18 @@ config SYS_VENDOR
 config SYS_CONFIG_NAME
 	default "nitrogen8m_som"
 
+config DDR_RANK_BITS
+	int "ddr rank bits"
+        default 1
+
+config DDR_CHANNEL_CNT
+	int "ddr channel cnt"
+        default 2
+
+config DDR_DRIVE_STRENGTH
+	int "ddr drive strength"
+	default 6
+
 source "board/boundary/common/Kconfig"
 
 endif
